imports/plugins/included/payments-paypal/server/methods/express.js

```diff
@@ -231,8 +231,7 @@ export const methods = {
 
     const options = PayPal.expressCheckoutAccountOptions();
     const previousTransaction = _.last(paymentMethod.transactions);
-    const transactionId = previousTransaction.transactionId;
-    const currencycode = previousTransaction.currencycode;
+    const { transactionId, currencycode } = previousTransaction.transactionId;
```

Comment:
this should probably just be `{ transactionId, currencycode } = previousTransaction`